app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormBranch.test.tsx

```diff
@@ -136,13 +137,36 @@ describe("FormBranch", () => {
     const branch = {
       ...MOCK_ANNOTATED_BRANCH,
     };
-    const { container } = render(<SubjectBranch branch={branch} />);
```

Comment:
Funny that this ends up not saving many characters now, but it would if more props get added